src/collision/cm_test.c

```diff
@@ -371,6 +372,12 @@ static void Cm_BoxLeafnums_r(cm_box_leafnum_data *data, int32_t node_num) {
 		const cm_bsp_node_t *node = &cm_bsp.nodes[node_num];
 		const cm_bsp_plane_t *plane = node->plane;
 
+		if (data->matrix) {
+			static __thread cm_bsp_plane_t rotated;
+			rotated = Cm_TransformPlane(*data->matrix, plane);
+			plane = &rotated;
+		}
```

Comment:
Why not just use a `cm_bsp_plane_t` instead of `cm_bsp_plane_t *`? Copy the node's plane into the var, and then transform the copy if needed. Then you don't need TLS here.